concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -12,28 +13,18 @@
 from utils.helpers          import get_current_utc_timestamp
 from utils.testing_helpers  import generate_ecc_key_pair
 
-from api_testing_helpers    import api_request
-from api_testing_helpers    import timestamp_to_isoformat
+from api_testing_common     import api_request, parse_command_line, get_protocol_constants, print_protocol_constants
+from api_testing_common     import timestamp_to_isoformat
 
-import requests
+HOUR = 3600
```

Comment:
I still think there is no need to keep that.

Reply:
@rwrzesien I still prefer to have it here ;)

Reply:
I also think it's not needed and I would not define it myself but it does not break anything so if Piotr prefers to keep it, I'm for keeping it. It looks like a matter of opinion to me.

Reply:
Off course, I was just wondering if this can get more meaningful name in context of the usage in test.